fud2/scripts/profiler.rhai

```diff
@@ -31,6 +32,61 @@ fn profiling_setup(e) {
     // Standalone Verilog testbench. copied from testbench
     e.rsrc("tb.sv");
 
+    // ---- eDSL to Calyx to flame graph.
+    e.rule("edsl-to-calyx", "python3 $in $edsl-args > $out");
+
+    e.var_("metadata-mapping-json", "metadata-map.json");
+    e.config_var_or("fileinfo_emitter", "fileinfo_emitter", "$calyx-base/target/debug/fileinfo_emitter");
+    e.rule("parse-metadata", "$fileinfo_emitter -l $calyx-base $in > $metadata-mapping-json");
+
+    e.rule("parse-vcd-from-adl", "python3 $parse-vcd-script $in $cells $metadata-mapping-json profiler-out $out");
+}
+
+// first pass. probably worth merging with calyx_to_flamegraph one of these days
```

Comment:
try to have a consistent style for `TODO` comments:
```
// TODO: merge with calyx_to_flamegraph
```

Reply:
Will keep in mind for the future!